### ISSUE TITLE
Bearer Token Is Saved To Workspace When Using Auth Module

### ISSUE THREAD

**Issue by baileylo** [a85/POSTMan-Chrome-Extension#6543]
**Describe the bug**

With a shared collection using the Authorization tool the bearer token is saved to the request for other people with access to the collection to get.

**To Reproduce**
1. Have a shared collection
3. Create an endpoint
4. Go to Authorization Tab 
5. Select OAuth 2.0 as the Type
6. In the Access Token field type in "fizzbuzz"
7. Click save
8. Teammate looks at Endpoint and sees "fizzbuzz" as the Access Token

**Expected behavior**
The access token should not be stored

**Screenshots**
![postman](https://user-images.githubusercontent.com/145345/57948117-a712d400-7895-11e9-85f4-5ab1b630d339.png)

**App information (please complete the following information):**
 - App Type Native App
 - Postman Version 7.1.0
 - OS: 10.14.5 

**Additional context**
You can add an environment variable in the input field and that works, but then that breaks the use of _Get New Access Token_ button as well as the _Available Token_ dropdown.

**Comment by shamasis** [comment]
The additional context you provided is the recommended way of sharing sensitive information. Can you elaborate on what is getting in the way of doing that? You can have a shared environment where teammates can put in their own tokens.

**Comment by baileylo** [comment]
@shamasis This is what I'm seeing:

Step 1: Set Auth For Route
<img width="965" alt="Screen Shot 2019-05-20 at 10 24 32 PM" src="https://user-images.githubusercontent.com/145345/58070512-2e8a6c80-7b4e-11e9-9432-b14ff8525ee4.png">

Step 2: Select an Existing OAuth Token
<img width="965" alt="Screen Shot 2019-05-20 at 10 24 41 PM" src="https://user-images.githubusercontent.com/145345/58070511-2e8a6c80-7b4e-11e9-92c7-a1dd10484522.png">

Step 3: See OAuth Token
<img width="961" alt="Screen Shot 2019-05-20 at 10 24 50 PM" src="https://user-images.githubusercontent.com/145345/58070510-2e8a6c80-7b4e-11e9-93d7-fe6911e98a16.png">

At this point if you save the request, my access token will be stored and shared. I don't want to share this access token. How do I not share it?

**Comment by saswatds** [comment]
Hi @baileylo, yours is an excellent use-case for using postman environments. Here is a blog post that explains how you can use environment variables within postman https://blog.getpostman.com/2017/12/29/10-tips-for-working-with-postman-variables/

It would look something like this
<img width="1134" alt="Screenshot 2019-05-21 11 06 59" src="https://user-images.githubusercontent.com/8796447/58070948-983f6080-7bb8-11e9-80d5-419952712fe8.png">

**Comment by justtonytone** [comment]
@shamasis @codenirvana @vkaegis if clicking on `Use Token` shares the bearer token with other people, is there an easier method to set the token in our environment?

Currently we need to copy and paste it, but it would be nice to add a button there to run a script that could automatically set an environment variable.

**Comment by shamasis** [comment]
Yep. We need to come up with a better flow. We have plans to rework on the oAuth flows. No ETA yet since we are at the phase where we are accumulating info.

**Comment by baileylo** [comment]
I'm interesting if this falls as a security bug and how you're protecting/securing access tokens for storage?

**Comment by ghost** [comment]
Is there an update on this bug getting fixed. Our team just migrated to the shared collections and our bearer tokens are being overwritten by the last person that updated the collection with a new token
